src/components/GlobalMap/ShipDetailsMap/ShipDetailsMapReducer.js

```diff
@@ -0,0 +1,30 @@
+import { SHIP_DETAILS_MAP } from '../../../actions/types';
+
+const INITIAL_STATE = {
+    footerHeight: '10%',
+    mapHeight: '90%',
+    isFooterSelectionOpen: false,
+};
+
+export default (state = INITIAL_STATE, action) => {
+    switch(action.type) {
+        case SHIP_DETAILS_MAP.TOGGLE_FOOTER:
+            let {footerHeight, mapHeight, isFooterSelectionOpen} = state;
+            if(state.isFooterSelectionOpen) {
+                footerHeight = '10%';
+                mapHeight = '90%';
+                isFooterSelectionOpen = !state.isFooterSelectionOpen;
+            } else {
+                footerHeight = '70%';
+                mapHeight = '30%';
```

Comment:
Two different components need to use the same state. Wanted to use redux for that